src/resources/elements/textfield.html

```diff
@@ -6,7 +6,7 @@
         <img src="res/info.svg" alt="Info"/>
       </tooltip>
     </label>
-    <input type="text" id.bind="id" list.bind="listID & attr" value.bind="value" placeholder.bind="placeholder">
+    <input type.bind="format" id.bind="id" list.bind="listID & attr" value.bind="value" placeholder.bind="placeholder">
```

Comment:
Can we fully spell `attribute` instead of `attr`? Generally, we should avoid mental mapping, meaning variable names should be descriptive and properly spelled.